cli/flox/src/commands/services/restart.rs

```diff
@@ -21,27 +30,89 @@ pub struct Restart {
 
 impl Restart {
     #[instrument(name = "restart", skip_all)]
-    pub async fn handle(self, flox: Flox) -> Result<()> {
+    pub async fn handle(self, config: Config, flox: Flox) -> Result<()> {
         subcommand_metric!("services::restart");
 
-        let env = supported_environment(&flox, &self.environment)?;
+        let mut concrete_environment = supported_concrete_environment(&flox, &self.environment)?;
+        let activated_environments = activated_environments();
+
+        if !activated_environments.is_active(&UninitializedEnvironment::from_concrete_environment(
+            &concrete_environment,
+        )?) {
+            return Err(anyhow!(indoc! {"
+                Cannot restart services for an environment that is not activated.
+
+                To activate and start services, run 'flox activate -s'
```

Comment:
should this be `--start-services` just to be less am biguous about what you want users to do?